models/prices/prices_usd_latest_old.sql

```diff
@@ -0,0 +1,18 @@
+{{ config(
```

Comment:
Can this be made incremental?

Reply:
Yeah I guess, it would make the runs a bit faster.